integration/modify_test.go

```diff
@@ -0,0 +1,34 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/naggie/dstask"
+	"gotest.tools/assert"
+)
+
+func TestModify(t *testing.T) {
+	t.Skip("modify test TODO")
```

Comment:
@naggie skipping this one also. What is the expected behavior for modify with respect to IDs and context?

Is it:

* 1 or more IDs always takes precedence, no matter what context is set
* if a context is set (and no IDs provided), a modify adds a tag or project to every task in the context?

What is an example of a modify command outside context? e.g. involving the `--`

Reply:
Yes, correct. I suppose naturally the `--` would work as if a context was not set -- i.e. if there are no IDs, set to all non-resolved tasks.

Reply:
I have added a test for modifying by IDs, but we will have to implement context-based modify in a later PR. A prompt is issued if an ID is not set:

```
--- FAIL: TestModify (0.06s)
    modify_test.go:31: stdout: 
    modify_test.go:31: stderr: No IDs specified. Apply to all 1 tasks in current ctx? [y/n] panic: EOF
```